compiler/ballerina-backend-jvm/src/main/ballerina/compiler_backend_jvm/jvm_type_gen.bal

```diff
@@ -372,6 +372,39 @@ function createObjectType(jvm:MethodVisitor mv, bir:BObjectType objectType, bir:
     return;
 }
 
+# Create a runtime type instance for the service.
+#
+# + mv - method visitor
+# + objectType - object type
+# + typeDef - type definition of the service
+# + pkgName - name of the module the service belongs to
+function createServiceType(jvm:MethodVisitor mv, bir:BObjectType objectType, bir:TypeDef typeDef, string pkgName) {
```

Comment:
we can unify this with object type right? only change is the new instruction and the init, (better to do that with a separate PR)